simple_salesforce/login.py

```diff
@@ -34,19 +36,28 @@ def SalesforceLogin(
     * security_token -- the security token for the username
     * organizationId -- the ID of your organization
             NOTE: security_token an organizationId are mutually exclusive
-    * sandbox -- True if you want to login to `test.salesforce.com`, False if
-                 you want to login to `login.salesforce.com`.
+    * sandbox -- DEPRECATED: Use login_domain instead.
     * sf_version -- the version of the Salesforce API to use, for example
                     "27.0"
     * proxies -- the optional map of scheme to proxy server
     * session -- Custom requests session, created in calling code. This
                  enables the use of requets Session features not otherwise
                  exposed by simple_salesforce.
     * client_id -- the ID of this client
+    * login_domain -- The domain to using for logging in. Use common login
+                      domains, such as 'login' or 'test', or named domain.
     """
-
-    soap_url = 'https://{domain}.salesforce.com/services/Soap/u/{sf_version}'
-    domain = 'test' if sandbox else 'login'
+    if 'sandbox' in kwargs.keys():
+        warnings.warn('\'sandbox\' argument is deprecated. Use '
```

Comment:
Super nitpick here, but rather than escaping, can you alternate single and double quotes? Whichever you prefer. Not merge blocking, I only find it more aesthetically pleasing when reading through code.